io_scene_dae/export_dae.py

```diff
@@ -615,12 +615,13 @@ def export_mesh(self, node, armature=None, skeyindex=-1, skel_source=None,
                         name = node.vertex_groups[vg.group].name
 
                         if (name in si["bone_index"]):
-                            # TODO: Try using 0.0001 since Blender uses
-                            #       zero weight
-                            if (vg.weight > 0.001):
-                                v.bones.append(si["bone_index"][name])
-                                v.weights.append(vg.weight)
-                                wsum += vg.weight
+                            # Using 0.001 since Blender uses zero weight
```

Comment:
Why is using zero weight here a problem? Is it a problem for Godot, for COLLADA? Pinging @reduz